release notes/v0.40.0.md

```diff
@@ -0,0 +1,83 @@
+k6 v0.40.0 is here! :tada: 
+#TODO 
+[Roadmap and future plans section](#roadmap-and-future-plans).
+
+## New Features!
+
+### experimental extensions
+https://github.com/grafana/k6/pull/2630
+https://github.com/grafana/k6/pull/2656
+
+
+### https://github.com/grafana/k6/pull/2571
+
+### First class support for JavaScript Classes [#]()
+https://github.com/grafana/k6/issues/1763
+https://github.com/grafana/k6/pull/2618
+https://github.com/grafana/k6/pull/2610
+Other goja updates 
+
+https://github.com/grafana/k6/pull/2645
+https://github.com/grafana/k6/pull/2600
+https://github.com/dop251/goja/pull/405
+
+### New Test runtime for module extension developers
+
+https://github.com/grafana/k6/issues/2598
+https://github.com/grafana/k6/pull/2602
+https://github.com/grafana/k6/pull/2621
+
+
+### https://github.com/grafana/k6/pull/2616
+
+
+## Breaking changes
+
+- [#2632](https://github.com/grafana/k6/pull/2632) As part of refactor to set tags provided to `metric.add` in the order they are provided, it was found out that you can provide tags as multiple objects/maps. This has never been the intend use and has never been documented. As such and as that makes no sense given every other API we have decided to remove this ability.
+- [#2582](https://github.com/grafana/k6/pull/2582) If `RegisterCallback` result is called twice this will now lead to a panic, instead of the second call silently breaking the event loop. This has never been the expected behaviour and calling it twice is always a bug in the code using it.
+- As part of [#2596](https://github.com/grafana/k6/pull/2596) the `tainted` property of the Metric type is no longer outputted by the JSON output. That property was (likely) always going to have the value `false` as it was outputted in the beginning of the test.
```

Comment:
```suggestion
- [#2632](https://github.com/grafana/k6/pull/2632) As part of refactoring to set tags provided to `metric.add` in the order they are provided, it was found out that you can provide tags as multiple objects/maps. This has never been the intend use and has never been documented. As such and as that makes no sense given every other API we have decided to remove this ability.
- [#2582](https://github.com/grafana/k6/pull/2582) If `RegisterCallback` result is called twice this will now lead to a panic, instead of the second call silently breaking the event loop. This has never been the expected behavior and calling it twice is always a bug in the code using it.
- As part of [#2596](https://github.com/grafana/k6/pull/2596) the `tainted` property of the Metric type is no longer outputted by the JSON output. That property was (likely) always going to have the value `false` as it was outputted at the beginning of the test.
```